app/javascript/components/toolbar/tests/toolbar.test.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
```

Comment:
Umm, I think this should be moved consistently to the place where any other test is for components...